client/checkout/classic/index.js

```diff
@@ -0,0 +1,189 @@
+/* global jQuery */
+
+/**
+ * Internal dependencies
+ */
+import { PAYMENT_METHOD_NAME } from '../constants.js';
+import { getConfig } from './../utils.js';
+import WCPayAPI from './../api';
+
+jQuery( function( $ ) {
+	// Create an API object, which will be used throughout the checkout.
+	const api = new WCPayAPI( {
```

Comment:
Generally the script should not be enqueued without the checkout form, however I agree with you. I added a check whether the configuration is available in 271d448, so that no JS errors would be present.